docs/network-mobile/mobile-proof-of-coverage.mdx

```diff
@@ -270,17 +266,17 @@ L.A. market area.
 
 :::tip check coverage
 
-Modeled Coverage Explorer is available [here](https://planner.hellohelium.com/).
+Modeled Coverage Explorer is [available](https://planner.hellohelium.com/).
```

Comment:
Should explain the link a bit more. Perhaps something like:
"View mobile Hotspot coverage on the [modeled coverage explorer]"